src/store/index.js

```diff
@@ -0,0 +1,16 @@
+import Vue from "vue";
+import Vuex from "vuex";
+import actions from "./actions";
+import mutations from "./mutations";
+
+Vue.use(Vuex);
+
```

Comment:
This file is the heart of the store

Imports Vue, Vuex, actions and mutations, and tells the Vue instance to use Vuex